src/test/java/com/beust/jcommander/DefaultUsageFormatterTest.java

```diff
@@ -146,6 +146,31 @@ class ArgsTemplate {
         Assert.assertEquals(sb.toString(), expected);
     }
 
+    @Test
+    public void testPlaceholder() {
+        class ArgsTemplate {
+            @Parameter(names = {"-i"}, placeholder = "<filename>")
+            public String inputFilename;
+        }
+
+        // setup
+        StringBuilder sb = new StringBuilder();
+        JCommander jc = JCommander.newBuilder()
+                .addObject(new ArgsTemplate())
+                .build();
+
+        // action
+        jc.getUsageFormatter().usage(sb);
+
+        // verify
+        String expected = "Usage: <main class> [options]\n"
```

Comment:
* Feel free to use `"""` text blocks, if you like. We are on Java 17 in this branch. 😃

Reply:
Done